options.h

```diff
@@ -67,6 +69,10 @@ class FlayOptions : public AbstractP4cToolOptions {
 
     /// Toggle use of placeholder variables to model recirculated or cloned packets.
     bool usePlaceholders_ = false;
+
+    /// In strict mode, Flay will report error upon adding more reachability condition for existing
+    /// nodes
```

Comment:
```suggestion
    /// In strict mode, Flay will report errors instead of warnings for certain unsafe behavior.
    /// For example,  when adding more than one reachability condition for one IR node. 
```